src/test/java/com/google/sps/CommentAnalysisTest.java

```diff
@@ -232,5 +255,111 @@ public void testDistributeScore() {
     Assert.assertEquals(
         constructSentimentBucketListFromCommentList(expectedUserComment, expectedFrequency),
         distStat.getSentimentBucketList());
+    Map<String, Integer> expectedMap =
+        new HashMap<String, Integer>() {
+          {
+            put("1", 2);
+            put("comment", 2);
+            put("third", 1);
+            put("forth", 1);
+            put("pos", 1);
+            put("neg", 1);
+          }
+        };
+    Assert.assertEquals(expectedMap, distStat.getWordFrequencyMap());
```

Comment:
This addition is not consistent with the test name or the test case. We could remove this.